posts/rust-wasm.md

```diff
@@ -33,24 +34,24 @@ and it's going to be around for a very long time.
 
 Alright, so why is it a big deal?
 
-1. It's a binary portable format supported in something everyone has, a web
+1. It's a portable binary format supported in something everyone has, a web
    browser. It's very much like Java applets many years ago.
 2. It's platform independent. Mac, Linux, Windows, whatever OS it doesn't matter
    as long as it has a web browser.
-3. It's an open standard, so no one company owns it like Oracle does with Java.
-4. It's fast, while JavaScript is great due to it's ubiquity and working
+3. It's an open standard, so no one company owns it (like Oracle does with Java).
+4. It's fast. While JavaScript is great due to it's ubiquity and working
    everywhere on the web, it's interpreted so it has it's limits with how fast
```

Comment:
honestly, I would say "its semantics" over "it's interpreted", because as you note, JS is often JIT compiled. interpretation isn't the root cause of why it's slower; wasm can be interpreted too.